spec/text-buffer-spec.coffee

```diff
@@ -1183,15 +1183,15 @@ describe "TextBuffer", ->
         expect(calls).toBe(0)
 
       it "changes the in-memory contents of the buffer to match the new disk contents and notifies ::onDidChange observers", ->
-        expect(event1.oldRange).toEqual [[0, 0], [0, 0]]
```

Comment:
It's actually nice that the order is delete-then-insert now. Makes more sense to me intuitively.